src/index_spec.js

```diff
@@ -0,0 +1,134 @@
+const sinon = require('sinon');
+const Uploader = require('./index.js');
+const FileAPI = require('file-api')
+const {expect} = require('chai');
+const {describe} = require('mocha');
+
+global.FormData = require('form-data');
```

Comment:
Some of the describes can go in another files. Some of them don't really use the new file you create and don't need the same beforeEach

Reply:
You're right, fixed it 👍